README.md

```diff
@@ -1,5 +1,7 @@
 ## Django-invitations - Generic invitations app
 
+[WORKING ON PATCH FOR DJANGO 3.0 COMPATIBILITY]
+
```

Comment:
Why is this in the README?

Reply:
My bad. Had that in the Test branch before doing the merge into the master branch.  Will remove.